libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -164,7 +164,11 @@ struct eosvmoc_tier {
             }
          }
 #endif
-         const bool allow_oc_interrupt = attempt_tierup && context.is_applying_block() && context.trx_context.has_undo();
+         // do not allow oc interrupt if no undo as the transaction needs to be undone to restart it.
+         // do not allow oc interrupt if implicit as deferred trx onerror execute as implicit outside the transaction
```

Comment:
This wording `if implicit as deferred trx` is a little confusing. Implicit trx and deferred trx are different.

Reply:
I reworked the comment. https://github.com/AntelopeIO/spring/pull/1150/commits/845576c52b8cabecf6b315dbc44988debd6b1924